.gitignore

```diff
@@ -52,10 +52,6 @@ vtest/Pictures
 dependencies
 
 # Downloaded files during build process
-MuseScore_General.sf3
-MuseScore_General_License.md
-MuseScore_General_Changelog.txt
-MuseScore_General_Readme.md
```

Comment:
Now that we are adding the sound font to the repo, could we also remove the downloading from the build process? (i.e. DownloadSoundFont.cmake etc.)

Reply:
I'm not sure whether adding a ~80MB binary file to the repo is a good idea. A couple years back we got rid of that and for IMHO good reason

Reply:
> Now that we are adding the sound font to the repo, could we also remove the downloading from the build process? (i.e. DownloadSoundFont.cmake etc.)

let's keep them, they shouldn't run if there is already a sound-font in the "share" folder

Reply:
DownloadSoundFont.cmake rather compares the contents of the share/sound/SF_VERSION file with https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/VERSION.